src/data/template/Microsoft.Authorization/policyAssignments.template.jq

```diff
@@ -1,6 +1,12 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
+    "metadata": {
+        "_generated": {
```

Comment:
To align with the schema used in [bicep](https://github.com/Azure/bicep/issues/800), the property should be `_generator` and not `_generated`. Please change that for all instances. 

![image](https://user-images.githubusercontent.com/16622613/131856602-a5dbc966-5131-4b1a-b971-38d4aab6f855.png)

Reply:
> To align with the schema used in [bicep](https://github.com/Azure/bicep/issues/800), the property should be `_generator` and not `_generated`. Please change that for all instances.
> 
> ![image](https://user-images.githubusercontent.com/16622613/131856602-a5dbc966-5131-4b1a-b971-38d4aab6f855.png)
Agreed Johan, have removed the version and updated the metadata name to _generator.